queue/management/commands/run_consumer.py

```
@@ -63,10 +63,9 @@ def assign_queues_to_workers(num_workers, queue_assignments):
 
 
 class Command(BaseCommand):
-    args = "<worker count>"
```

I realize this appears unused, but the default execution of this to start the second part of the xqueue app passed in a worker count
https://github.com/edx/configuration/blob/master/playbooks/roles/xqueue/templates/xqueue_consumer.conf.j2#L3
I think the right fix is just to pull that out of the supervisor runner for xqueue, but I'm trying to make sure it's really unused

---

It was probably used in previous versions of the code, but was silently ignored by the code in master before this PR.  The entire `assign_workers_to_queues` and `assign_queues_to_workers` functions seem to have been similarly orphaned.  Those should be deleted if we don't have immediate plans to use them again, but that was a little out of scope for this particular PR.

---

Agreed that we can probably delete more of it - I'd been trying to figure out how it was being used in production, but hadn't chased through it all yet.  I only noticed because the consumer stopped starting because it passed a count that now throws an error.  I fixed that on devstack but suspect it needs a broader set of deletes.